data/technicave.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Aryan Tiwari",
+  "type": "personal",
+  "bio": "| Full Stack Developer | Chief Creative Officer at Showwcase | Creator at Technicave | Loves to Contributing to the Community |",
+  "avatar": "https://github.com/technicave.png",
+  "links": [
+    {
+      "name": "Connect on Twitter",
+      "url": "https://twitter.com/technicave",
+      "icon": "FaTwitter"
+    },
+    {
+      "Name": "Checkout Github",
+      "url": "https://github.com/technicave",
+      "icon": "FaGithub"
+    },
+    {
+      "Name": "Follow on LinkedIn",
+      "url": "https://www.linkedin.com/in/technicave/",
+      "icon": "FaLinkedin"
+    },
```

Please remove the extra **' Comma '** from this line :+1: 
```suggestion
    }
```